peer/peerlist/v2/list_test.go

```diff
@@ -222,3 +223,18 @@ func TestPeerList(t *testing.T) {
 		},
 	}))
 }
+
+func TestFailFast(t *testing.T) {
+	fake := yarpctest.NewFakeTransport(yarpctest.InitialConnectionStatus(peer.Unavailable))
+	impl := &mraList{}
+	list := New("mra", fake, impl, FailFast())
+
+	ctx, cancel := context.WithTimeout(context.Background(), testtime.Second)
```

Comment:
Should we use `time.Second` and ensure that this returns in under 1s or a couple hundred milliseconds?